nss-day-cohort-48/giffygram-webcrawlers
# Dmg edit profile

#### Changes Made
1, added some user profile edit functionality
2. fixed issue with post likes not deleting properly
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout dmg-edit-profile
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   Log in and like/unlike posts and make sure that they are adding and deleting from the database properly
